packages/schematics/angular/service-worker/config/schema.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
I am not sure if the schema should be here in the first place. I think the right place for this would be in `@angular/service-worker`. As really the schema can be used by anyone who uses that package, and doesn't necessary have a dependency on `@schematics/angular`. Also, it will ease maintenance, as it's harder to keep models in sync within different repos. 

@gkalpak, what do you think?

Reply:
I was also not sure of the place of the schema. Just waiting a green light, and I'll do the PR in Angular repo.

Reply:
Done in angular/angular#27859